deploy/infra-cluster-service-account.yaml

```diff
@@ -29,4 +29,4 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: kubevirt-csi
----
\ No newline at end of file
+---
```

Comment:
I think you can remove this file, you split the rbac into two below.

Reply:
this is still needed for the use case where both controller and node are deployed in tenant cluster.